### ISSUE TITLE
Icons are broken for me since latest update

### ISSUE THREAD

**Issue by jclsn** [lambdalisue/fern-renderer-nerdfont.vim#26]
The icons are broken since the last update

![image](https://github.com/lambdalisue/fern-renderer-nerdfont.vim/assets/19779511/b864d652-ff80-42b4-a523-670f81c760a0)

Some are too small, some have a prepended letter and some have double icons

Seems like this commit has broken it 

https://github.com/lambdalisue/fern-renderer-nerdfont.vim/commit/e472ea1b57c39a651fe0fee3907fbfc5ad296446

**Comment by lambdalisue** [comment]
Did you update https://github.com/lambdalisue/nerdfont.vim? If you didn't, just update it. If you did and somehow it break, please find a minimal reproduction steps (including information about OS, Vim, Terminal, Shell, and vimrc)

**Comment by jclsn** [comment]
All plugins got updated together

OS: Manjaro
Vim: 9.0 1-2143
Terminal: Kitty, Alacritty, Konsole
Shell: zsh, bash
vimrc: I could strip my .vimrc down to this and the problem still occurs
```vim
call plug#begin('~/.vim/plugged')
        Plug 'lambdalisue/fern.vim'
        Plug 'lambdalisue/fern-renderer-nerdfont.vim'
        Plug 'lambdalisue/glyph-palette.vim'
        Plug 'lambdalisue/nerdfont.vim'
call plug#end()

let g:fern#renderer = "nerdfont"

nnoremap <silent> <C-n> :Fern . -drawer -toggle<CR>
```

**Comment by jclsn** [comment]
After opening the fern buffer even my terminal is broken and the cursor is below the home button. Only when I use zsh with p10k there though. No issues with bash. I have now downgraded ``fern.vim``, ``fern-nerdfont-renderer.vim`` and ``nerdfont.vim`` as far as I can and now all my problems are gone.

I have now frozen all of your plugins. Please let me know when you think you have found a fix and I am happy to test it.

**Comment by lambdalisue** [comment]
@jclsn

It seems that this issue is environment-dependent, making reproduction very challenging. However, since the behavior of something that was previously working is now disrupted, we have changed the approach of the relevant feature of the nerdfont from opt-out to opt-in, as it goes against the plugin-providing policy. The fern-renderer-nerdfont.vim has been reverted to its original state.

Thank you for providing detailed feedback.
